COLLABORATOR_GUIDE.md

```diff
@@ -345,7 +345,7 @@ Additionally:
 
 ### Technical HOWTO
 
-_Optional:_ ensure that you are not in a borked `am`/`rebase` state
+Optional: Clear any `am`/`rebase` that is already underway.
```

Comment:
This isn't really `optional`, it's necessary if you are in an `am` or `rebase` state. People who are less familiar with Git aren't going to know whether or not they need to do it.

Maybe say something like:

>If `git status` shows `Rebase in progress` or `am in progress`, then you need to cancel the `rebase` or `am` with: